featuremanagement/_featuremanager.py

```diff
@@ -37,8 +37,8 @@ def __init__(self, configuration, **kwargs):
                 raise ValueError("Custom filter must be a subclass of FeatureFilter")
             self._filters[feature_filter.name] = feature_filter
 
-    @overload
-    def is_enabled(self, feature_flag_id, user_id, **kwargs):
+    @overload  # type: ignore
+    def is_enabled(self, feature_flag_id: str, user_id: str, **kwargs: Dict) -> bool:
```

Comment:
May I know the reason why here you have `**kwargs: Dict`, but for `FeatureFilter.Evaluate` you have `**kwargs: Unpack[RequestParams]`?

Reply:
I'm actually unsure which one it should be. I imagine it's should be a `Dict[str, Any]` but I couldn't find any good documentation on it.